domains/auth/Auth.API/API/Values/UserScopeClaim.cs

```diff
@@ -0,0 +1,11 @@
+namespace API.Values;
+
+public struct UserScopeClaim
```

Comment:
These are more `UserScopeName` than `UserScopeClaim` since they are the values inside of a claim.